sof-guest-defconfig

```diff
@@ -0,0 +1,16 @@
+CONFIG_VIRTIO_BLK=y
+CONFIG_VIRTIO_NET=m
+CONFIG_VIRTIO_CONSOLE=y
+CONFIG_HW_RANDOM_VIRTIO=m
+CONFIG_DRM_VIRTIO_GPU=m
+CONFIG_SND_SOC_INTEL_SKL_HDA_DSP_GENERIC_MACH=n
+CONFIG_SND_SOC_SOF_NOCODEC_SUPPORT=y
```

Comment:
that's surprising. Can you explain this dependency?
with this option, you create a card that has a set of SSP and DMIC back-end dailinks, not sure how relevant it is for virtio guests?

Reply:
The guest uses the NoCodec driver for its setup: it calls `sof_nocodec_setup()`. Yes, I know that nocodec cannot be enabled in production configurations, so, this has to be resolved somehow. That's why I was poking at allowing the NoCodec driver to co-exist with the rest with SOF. I mentioned this dependency before in the main VirtIO PR thread.